cetmix_tower_server/models/cx_tower_plan.py

```diff
@@ -182,6 +182,7 @@ def _get_next_action_values(self, command_log):
                                 "variable_id": variable.id,
                                 "value_char": variable_value.value_char,
                                 "server_id": server.id,
+                                "name": variable_value.value_char,
```

Comment:
Why do we need it? And why `variable_value.value_char` is used for the "name" field value?

Reply:
Thanks, I'll fix it